tests/locustfile.py

```diff
@@ -0,0 +1,74 @@
+from random import randrange
+from typing import List
+
+from locust import HttpUser, task
+from matchms import Spectrum
+from matchms.importing import load_from_mgf
+
+from config import ENV
+from omigami_client import Spec2VecClient
+from omigami_client.spec2vec import JSON
+from tests.conftest import ASSETS_DIR
+
+# initial randomized payload setup
+
+# settings
+MAX_DIFFERENT_BATCHES = 5
+BATCH_SIZE = 1000
+TOKEN = ENV["token"].get()
+MGF_PATH = str(ASSETS_DIR / "GNPS-NIST14-MATCHES.mgf")
+PREDICT_ENDPOINT = (
+    "https://mlops.datarevenue.com/seldon/seldon/spec2vec/api/v0.1/predictions"
+)
+
+# spec2vec client
+spec2vec = Spec2VecClient("no-need-for-token-here")
```

Comment:
why don't we need the admin token for the dev cluster here ? Did we disable auth for the prediction endpoint on the dev cluster ?

Reply:
Its used in the post request itself, because here we don't use the client's post but the locust post

Reply:
So, we actually use the token just not here : ]